### DISCUSSION TITLE
How to set HyperHDR so first LED to light is 50th LED on strip

### DISCUSSION THREAD

**Discussion by DastardlyDino** [awawa-dev/HyperHDR#767] [Q&A] [answered]
I have an extra long LED strip that I do not want to cut and solder yet. I would like to use the initial 50 LEDs essentially as just wiring so my Dig-octa running WLED is not dangling in the air. Is there anyway to set up HyperHDR so that the first LED is actually the 50th on the strip?

**Comment by awawa-dev** [comment, marked as answer]
Yes, you can design something like this in HyperHDR. First, design the LED strip geometry using the standard way for the entire strip with these 50 redundant LEDs ( https://github.com/awawa-dev/HyperHDR/wiki/Quick-start#configuring-the-led-strip ) so that the 50th LED overlaps the position with the first LED which starts already on the TV. Then you need to right-click on the first 50 LEDs and select the 'disable' option (https://github.com/awawa-dev/HyperHDR/wiki/Quick-start#manual-led-layout-and-testing) then they will not be used. Optionally, instead of right-clicking, you can also manually add this property to the raw JSON in 'advanced settings' in the editor available on the same page where you configure the LED strip geometry.

**Reply by DastardlyDino** [reply]
I cannot seem to find the raw JSON where I believe you said it is. When I got to the advanced settings on the LED Layout page (http://YOUR.IP.ADDRESS.HERE:8090/#light_source) all I see is the attached image.

![image](https://github.com/awawa-dev/HyperHDR/assets/47339617/b4c618dd-51a8-4142-a9b4-ee9f987cf87e)

**Reply by awawa-dev** [reply]
Sorry it is below advanced settings:
![obraz](https://github.com/awawa-dev/HyperHDR/assets/69086569/3d0d3379-e4e6-48f7-b16d-549176a8b488)
If you dont see it and using HyperHDR v19 you may need to switch to advanced settings in the general configuration. In HyperHDR v20beta it is visible by default.

**Reply by awawa-dev** [reply]
And that how it looks when the LED is disabled:

<img src="https://github.com/awawa-dev/HyperHDR/assets/69086569/cfe0b03d-2f61-48f5-985c-3052ea14f26c" width="400px" />
